src/Microsoft.ML.Core/Data/IHostEnvironment.cs

```diff
@@ -72,6 +73,11 @@ public interface IHostEnvironment : IChannelProvider, IProgressChannelProvider
         /// The use of suffix and prefix, including whether they have any affect, is up to the host environment.
         /// </summary>
         IFileHandle CreateTempFile(string suffix = null, string prefix = null);
+
+        /// <summary>
+        /// Get the MEF composition container.
+        /// </summary>
```

Comment:
I kind of feel like this is not really a sufficient description for what it is for, or what it is used for?